src/Game.jsx

```diff
@@ -0,0 +1,37 @@
+import "./Game.css";
+import { useGameStateStore } from "./store";
+export function Game() {
+  const gameState = useGameStateStore((state) => state.gameState);
+  const move = useGameStateStore((state) => state.move);
+  const restart = useGameStateStore((state) => state.restart);
+  const loading = useGameStateStore((state) => state.loading);
+  console.log(loading);
+  return (
+    <div className="Game">
+      <p className="Game-description">{gameState.description}</p>
+      <button
+        onClick={() => {
+          restart();
+        }}
+        className="Game-restart"
+      >
+        Restart
+      </button>
+      <ul className="Game-actions">
+        {gameState.actions.map((action, index) => (
+          <li key={index}>
+            {/* here i made it so that the buttons are not clickble when loading */}
+            <button
+              // ${action.direction} is a template literal, it will be replaced with the value of action.direction
+              className={`Game-navigate ${action.direction}`}
+              disabled={loading}
```

Comment:
Very nice! As you know, one of the requirements this week is:
- Focus on making the UX of your app good. Handle the response delay

and I think it would be very nice with some kind of loading indicator, but on the other hand you have actually handled the response delay by disabling the buttons, so 👍